Mathlib/Order/SuccPred/Limit.lean

```diff
@@ -65,26 +65,26 @@ smaller.
 
 This previously allowed the element to be minimal. This usage is now covered by `IsSuccPrelimit`. -/
 def IsSuccLimit (a : α) : Prop :=
-  ¬ IsMin a ∧ IsSuccPrelimit a
+  ¬IsMin a ∧ IsSuccPrelimit a
 
-protected theorem IsSuccLimit.not_isMin (h : IsSuccLimit a) : ¬ IsMin a := h.1
```

Comment:
Please don't mix in a style change like this with a refactor PR.

Reply:
Done.